d2l-typography-shared-styles.html

```diff
@@ -26,6 +26,14 @@
 					text-transform: uppercase;
 				};
 
+				--d2l-body-compact-text: {
+					color: var(--d2l-color-ferrite);
+					font-family: inherit;
+					font-size: 0.8rem;
+					font-weight: 400;
+					line-height: 1rem;
```

Comment:
The place where I think I ran into issues with line-height might have been in an element that had overflow hidden.  I am pretty sure it was the menu-items types.